### ISSUE TITLE
some testing issue

### ISSUE THREAD

**Issue by jdily** [BartolomeD/domain_adapt#1]
Hi 
thx for writing this DANN code, I recently am also interested in this direction so I downloaded your code and try to test on MNIST/MNISTM sets.
I saw your post in pytorch forum, and you mentioned you got great performance.
Can you let me know what kind of parameter, e.g. learning rate you used to get those performance?
So far, I keep getting weird performance, i.e. < 50% on this pair.

thx

**Comment by dbrojas** [comment]
Hi jdily,


The DANN model as defined here in the repo was designed to work on the fog data. So the reason it doesn't work is probably due to both parameters and architectural decisions.

With the DANN model, I was able to replicate results for the USPS/MNIST experiment. This experiment uses a way simpler architecture to start with.

I am going to shift repository focus from the fog data to handwritten digits (MNIST, MNIST-M and USPS), since that data is publicly available anyway. 

Will keep you posted for the update.


Daniel

**Comment by jdily** [comment]
Hi Daniel, 
Thx for the quick reply.
I also feel like the arch in the current code is too complicated for MNIST series :)
And thx for planning shift the focus

ichao

**Comment by dbrojas** [comment]
Hi ichao,

I have updated the repo so the models are now tuned for the image datasets. 

Daniel
